Network Class/Network.h

```diff
@@ -18,55 +20,161 @@ typedef std::vector<int> Vector;
 
 class Network
 {
-	public:
-							Network				(istream& = cin);
-							Network				(const string&, const string&);
-							Network				(const string&);
-							~Network			();
-		void				train				();
-		void				classify			(const string&);
-
-	private:
-		double				learningRate;
-		int					batchSize;
-		int					epochs;
-		int 				numLayers;
-		
-		VMatrix				weights;
-		VMatrix				sumNablaW;
-		VMatrix				biases;
-		VMatrix				activations;
-		VMatrix				weightedInputs;
-		VMatrix				errors;
-		VMatrix				sumNablaB;
-		
-		Vector				miniBatchIndices;
-		Vector				layerSizes;
-	
-	//we do need the string names to be the class data members. Just realized, we need to write the names of the files in the Previous_Network_File.
-		string				trainingDataFilename;
-		string				expectedValuesFilename;
-	
-		ifstream			trainingDataInfile;
-		ifstream			expectedValuesInfile;
+public:
+
+	//**********Constructors and Destructors*******************
+						Network();
+						Network(const string&, const string&);	
+						Network(const string&);					
+						~Network();
+
+	//**********Public Accessible Methods**********************
+	void				forwardProp(int, ifstream &);
+	int					SGD();
+	std::vector<double>	train();
+	void				classify(const string&);
+	bool				writeToFile() const;
+	bool				backProp(int);
+
+	//***********Methods that read the values for the class*****
```

Comment:
Comment could be more specific, "the values" could literally be anything.

Reply:
Maybe, "Methods that read numLayers, layerSizes, epochs, learningRate, batchSize from file or console input"

Reply:
I agree. I just updated it.